src/subprocess_test.cc

```diff
@@ -261,3 +261,17 @@ TEST_F(SubprocessTest, ReadStdin) {
   ASSERT_EQ(1u, subprocs_.finished_.size());
 }
 #endif  // _WIN32
+
+#ifndef _WIN32
+TEST_F(SubprocessTest, Background) {
+  time_t before = time(NULL);
+  Subprocess* subproc = subprocs_.Add("echo -n 123; sleep 10s&");
```

Comment:
Ah, if this only happens on the failure case that's much better. But shouldn't 1s still be more than enough on even very slow systems?